src/stories/flows/eligibility/content/actions.flow.ts

```diff
@@ -0,0 +1,60 @@
+/*
+  eslint-disable max-len,
+                sonarjs/no-duplicate-string,
+ */
+import { IButton }    from '../../../../survey';
+import { TActionMap } from '../lib/contentMap';
+
+const online: Partial<IButton> = {
+  id:    'ab1',
+  link:  'https://www.ssa.gov',
```

Comment:
Did you want to keep ssa references out of the open source project?

Same goes for using phone numbers below.

The `example.com` domain is always safe to use. Also any phone number with 8675309 is blocklisted.